src/aiida/cmdline/params/options/main.py

```diff
@@ -592,6 +593,13 @@ def set_log_level(_ctx, _param, value):
     help='Only include entries created before OLDER_THAN days ago.',
 )
 
+MOST_RECENT_NODE = OverridableOption(
+    '-M',
+    '--most-recent-node',
+    is_flag=True,
+    help='Select the most recently created node.',
```

Comment:
Is there a plan to use this elsewhere for non-process nodes? If not, this help message could be more concrete. But I don't feel strongly, it's fine as is.

Reply:
I left it intentionally generic indeed. Maybe it could be used for `verdi calcjob` commands as well, like `verdi calcjob report` and `verdi calcjob gotocomputer`